exec/cfg.c

```diff
@@ -788,6 +845,48 @@ static void message_handler_req_exec_cfg_reload_config (
 	LEAVE();
 }
 
+/* Handle the phases of crypto reload
+ * The first time we are called is after the new crypto config has been loaded
+ * but not activated.
+ *
+ * 1 - activate the new crypto configuration
+ * 2 - clear out the old configuration
+ */
+static void message_handler_req_exec_cfg_reload_crypto (
+        const void *message,
+        unsigned int nodeid)
+{
+	const struct req_exec_cfg_reload_crypto *req_exec_cfg_reload_crypto = message;
+
+	log_printf (LOGSYS_LEVEL_DEBUG, "Crypto reconfiguration phase %d", req_exec_cfg_reload_crypto->phase);
+
+	if (req_exec_cfg_reload_crypto->phase) {
```

Comment:
Do we expect phase will ever be zero? If not I would suggest to remove this check